Chapters/04-implicitni-funkce.tex

```diff
@@ -91,5 +168,73 @@ \subsection{Věty o implicitní funkci}
 	\[ \mathbf{F}(\mathbf{x}, \mathbf{y}) = 0 \]
 \end{theorem}
 
+\begin{proof}
```

Comment:
Maybe it would make more sense to use the `intuition` or `example` block instead?